workflows/iuf/operations/management-nodes-rollout/management-worker-nodes-rollout.yaml

```
@@ -223,9 +250,20 @@ spec:
                     echo "NOTICE This step received no nodes to rebuild. Continuing without rebuilding any worker nodes"
                     exit 0
                   fi
-                  prepare_images_ouput=$(echo '{{inputs.parameters.global_params}}' | jq -r '.stage_params["prepare-images"].global["prepare-management-images"]["sat-bootprep-run"].script_stdout')
-                  image=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].final_image_id' | tr -d '"')
-                  configuration=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].configuration' | tr -d '"')
+
+                  cfs_config="{{=jsonpath(inputs.parameters.global_params,'$.input_params.cfs_configuration_management')}}"
+                  ims_image="{{=jsonpath(inputs.parameters.global_params,'$.input_params.boot_image_management')}}"
+
+                  if [[ -n $cfs_config && -n $ims_image ]]; then
+                    echo "INFO Rebuilding $rebuild_set using passed parameters $ims_image and $cfs_config"
+                    image=$ims_image
+                    configuration=$cfs_config
+                  else
+                    echo "INFO Starting rebuild of $rebuild_set using image built in the prepare images stage"
+                    prepare_images_ouput=$(echo '{{inputs.parameters.global_params}}' | jq -r '.stage_params["prepare-images"].global["prepare-management-images"]["sat-bootprep-run"].script_stdout')
+                    image=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].final_image_id' | tr -d '"')
+                    configuration=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].configuration' | tr -d '"')
```

Use `jq -r`, not `tr -d`
```suggestion
                    image=$(echo $prepare_images_ouput | jq -r '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].final_image_id')
                    configuration=$(echo $prepare_images_ouput | jq -r '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].configuration')
```